package.json

```diff
@@ -38,6 +38,7 @@
     "@adobe/openwhisk-loggly-wrapper": "^0.3.0",
     "ajv": "^6.5.4",
     "callsites": "^3.0.0",
+    "clone": "^2.1.2",
```

Comment:
Not needed: you can use `Object.assign({}, node)` instead.

Reply:
I chose `clone` over `JSON.parse(JSON.stringify(obj))` or `Object.assign({}, obj)` because of potential problems with complex objects (such as `Date`)... but if in a node all we expect to be cloned are strings, numbers and simple objects, we can drop it again.

Reply:
Fixed in https://github.com/adobe/hypermedia-pipeline/pull/120/commits/f74f8695efaaaa0ca1ae80cdbbae6cccf81ab4bb